idb/postgres/postgres.go

```diff
@@ -554,9 +573,9 @@ func (db *IndexerDb) yieldTxnsThread(ctx context.Context, rows *sql.Rows, result
 }
 
 // YieldTxns is part of idb.IndexerDB
-func (db *IndexerDb) YieldTxns(ctx context.Context, prevRound int64) <-chan idb.TxnRow {
+func (db *IndexerDb) YieldTxns(ctx context.Context, firstRound uint64) <-chan idb.TxnRow {
 	results := make(chan idb.TxnRow, 1)
-	rows, err := db.db.QueryContext(ctx, yieldTxnQuery, prevRound)
+	rows, err := db.db.QueryContext(ctx, yieldTxnQuery, int64(firstRound) - 1)
```

Comment:
would it make sense to also modify `yieldTxnQuery` to use `firstRound` directly?

Reply:
Yes, and I actually made an attempt to do it but there were subtle off-by-one issues so I decided not to tackle the problem in this PR.